runtime/io/build.gradle.kts

```diff
@@ -25,6 +25,7 @@ kotlin {
         commonTest {
             dependencies {
                 implementation(project(":runtime:testing"))
```

Comment:
I'm guessing we don't need this dependency in a lot of places anymore since _most_ usages of it were to pull in `runSuspendTest`. 

Not sure which projects are still using the other capabilities that remain.

Reply:
It seems that there are none, as I removed them all and nothing broke... :shrug:

Reply:
It seems that there are none, as I removed them all and nothing broke... :shrug: